website/docs/r/dns_managed_zone.markdown

```diff
@@ -53,3 +53,9 @@ DNS managed zones can be imported using the `name`, e.g.
 ```
 $ terraform import google_dns_managed_zone.prod prod-zone
 ```
+
+or by using `project-id` and `name` e.g.
+
+```
+$ terraform import google_dns_managed_zone.foo project/managedZones/foo-zone
```

Comment:
nitpick: could we maybe use `project-id` instead of `project`, to make it clear that `project` isn't a hardcoded string, and to match the description?

Reply:
Sorry for the delay. 
Sure. I documented all import formats like on compute disk using project-id